mllib/src/main/scala/org/apache/spark/ml/feature/StopWordsRemover.scala

```diff
@@ -73,22 +75,37 @@ class StopWordsRemover(override val uid: String)
   /** @group getParam */
   def getCaseSensitive: Boolean = $(caseSensitive)
 
-  setDefault(stopWords -> StopWordsRemover.loadDefaultStopWords("english"), caseSensitive -> false)
+  /**
+   * Locale for doing a case sensitive comparison
+   * Default: English locale ("en")
+   * @group param
+   */
+  val locale: Param[String] = new Param[String](this, "locale",
```

Comment:
Yes, but, How can we know that users loaded the French stopwords? User can load stopwords by
`StopWordsRemover.loadDefaultStopWords("french")`
and setting is
`new StopWordsRemover().setStopWords(stopWords)`
. Do you have any suggestion about that case?